src/components/Leaderboard/Leaderboard.jsx

```diff
@@ -28,7 +32,7 @@ export class Leaderboard extends React.Component {
                             <th><h3 className={styles.leaderboardTableColumn}>Username</h3></th>
                             <th><h3 className={styles.leaderboardTableColumn}>Score</h3></th>
                         </tr>
-                        {this.props.players.map(player => (
+                        {this.getSortedPlayers().map(player => (
```

Comment:
This way the list will be sorted every time a render happens, could we have them sorted earlier?